src/Domain.LinnApps/AuthorisedActions.cs

```diff
@@ -7,6 +7,8 @@ public class AuthorisedActions
         public const string CancelRequisition = "stores.requisitions.cancel";
 
         public const string BookRequisition = "stores.requisitions.book";
+
+        public const string ReverseRequisition = "stores.requisitions.reverse";
```

Comment:
So reversing is a differnet privilege than the usual function code one?

Reply:
yeah although hopefully you need the function code one too